app/schemas.py

```diff
@@ -58,7 +58,8 @@ class TransactionCreateSchema(Schema):
     @validates("amount")
     def validate_amount(self, value):
         if value < 0:
-            raise ValidationError("Amount is invalid")
+            self.error = ValidationError("Amount is invalid")
```

Comment:
Tu n'aura pas du toucher a ca surtout que ca n'a rien changé

Reply:
Oui, je n'ai pas touché à ça. Je sais pas de où le changement vient.

Reply:
Je l'ai revert to what it was  @komlan55